cluster-autoscaler/cloudprovider/gce/mig_info_provider.go

```diff
@@ -175,7 +173,7 @@ func (c *cachingMigInfoProvider) findMigWithMatchingBasename(instanceRef GceRef)
 }
 
 func (c *cachingMigInfoProvider) fillMigInstances(migRef GceRef) error {
```

Comment:
could we add some basic test for `fillMigInstances` testing different scenarios of when refresh should happen, and the values are being set.

Reply:
Added.